_docs/style/interactive-states.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Interactive States
+layout: docs
+category: Style
+usage: >
+  The system of interactive states ensures consistent styles for the states of controls in Helix. This system accounts for multiple types of controls, with various interactive patterns, and defines commonalities and standards across all of them.
+
+  All controls that live in the application body strictly adhere to subsets of the specifications in this document, unless otherwise noted in their individual design docs.
+resource: true
+status: complete
+last-modified: 2019-05-29
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+xxxxx.
```

Comment:
Use the interactive states styles anytime you're designing a new control or input that lives within the body of the application. Doing so will help keep your control aligned and consistent with Helix.

Reply:
Done.